### ISSUE TITLE
Error when re-download videos.

### ISSUE THREAD

**Issue by przrak** [OtchereDev/coursehunter_downloader#2]
Hello again)
I saw strange behavior. I downloaded some course. All fine. But when I decide download it again in the same folder, I expected, then program just matched files in folder and files in website. And nothing downloaded. But program downloaded some files, and wrote "update files". Very strange, because this will be same files.

**Comment by OtchereDev** [comment]
Wow that weird, I have gotten such an error before. 
The old video file were they downloaded with this same version of the downloader or the older version??

What is the course you trying it on??

**Comment by przrak** [comment]
Hello. I used same version twice.
I downloaded https://coursehunter.net/course/bazy-dannyh-chast-1 first time.
After that I decide start download again with "python main.py"
I think, that in second time app just validated my videos with videos on site and that is all. But some videos were re-downloaded. Not all videos.

**Comment by OtchereDev** [comment]
Oh I see what you are talking about, it not an error. By default, when you use the "All" download option it start downloading all the videos from 1 to the end but it first verifies that the video first exist on your computer. If does exist it will compare the size of both, if they are the same it skips but if the one on your computer is not of the same size as the one on the website it means it is not a complete video so the downloader will try to update it.

**Comment by OtchereDev** [comment]
To escaped the such behavior use the "range" download option, which will ask for the video range to be downloaded

**Comment by przrak** [comment]
No. First time I downloaded all videos. In second time I downloaded all videos again. This is the same videos for me. This videos not  updated. And in this situation "range" option can't help me. Because it is the same videos. I agree about range, when on ch added new videos in course, okey, in this situation I can use range and select missed videos.

**Comment by OtchereDev** [comment]
Okay cool but this is still weird, can you attach an images o f the situation??
And also do you have two different instance of the same video in the folder in which the course are downloaded into??

**Comment by przrak** [comment]
<img width="1389" alt="Screenshot 2021-04-02 10 36 03" src="https://user-images.githubusercontent.com/11232745/113408982-e2dab800-939f-11eb-9812-b1b63f2b4b0b.png">
I see, that video size changed. In first time it was 232 mb, and now it is 179 mb.
<img width="1926" alt="Screenshot 2021-04-02 10 43 08" src="https://user-images.githubusercontent.com/11232745/113409178-4369f500-93a0-11eb-9bb0-40c4517591a3.png">
Your app downloaded the same video file twice:( See last screenshot please. 3 and 4 file same size and duration, 5 and 6 file, 7 and 8 file, and e.t.c

**Comment by przrak** [comment]
Titles different, but when I see them, they equals.

**Comment by OtchereDev** [comment]
This is quite strange to me, first time encountering such an error but are you not sure it from coursehunter ??
